test/unit/instance_method_test.rb

```diff
@@ -226,35 +225,35 @@ def test_should_return_mock_for_stubbee
     mocha = Object.new
     stubbee = Object.new
     stubbee.define_instance_method(:mocha) { mocha }
-    method = ClassMethod.new(stubbee, :method_name)
+    method = InstanceMethod.new(stubbee, :method_name)
     assert_equal mocha, method.mock
   end
 
   def test_should_not_match_if_other_object_has_a_different_class
-    class_method = ClassMethod.new(Object.new, :method)
+    class_method = InstanceMethod.new(Object.new, :method)
```

Comment:
I think there are a bunch of places like this where the local variable, `class_method` should be renamed to `instance_method` or perhaps just `method`.

Reply:
Fixed now